radio/src/translations/en.h.txt

```diff
@@ -1027,6 +1027,9 @@
 #define TR_ANTENNACONFIRM2             "Make sure antenna is installed!"
 #define TR_LOWALARM                    INDENT "Low alarm"
 #define TR_CRITICALALARM               INDENT "Critical alarm"
+#define TR_RSSIALARM_WARN              TR("RSSI","TELEMETRY RSSI")
+#define TR_NO_RSSIALARM                "Audio alarms disabled"
```

Comment:
It actually only disables audio and Telemetry alarms disabled does not fit on X7. We would need to use Telemetry alarms dis. or something like that.